broker/broker_schema_mutator.go

```diff
@@ -0,0 +1,69 @@
+package broker
+
+import (
+	"github.com/uber/aresdb/metastore/common"
+	"github.com/uber/aresdb/metastore"
+	"github.com/uber/aresdb/memstore"
```

Comment:
depending on memstore will bring in unnecessary dependency
please rebase from newest change, david has moved memstore.TableSchema to common package

Reply:
+1 broker likely should only need to know schema and topology